subsys/bluetooth/services/ots/ots_oacp.c

```diff
@@ -161,6 +163,53 @@ static enum bt_gatt_ots_oacp_res_code oacp_delete_proc_validate(
 }
 #endif
 
+#if defined(CONFIG_BT_OTS_OACP_CHECKSUM_SUPPORT)
+static enum bt_gatt_ots_oacp_res_code oacp_checksum_proc_validate(
```

Comment:
This functions also calls the `obj_cal_checksum` callback, so it seems to do more than just validate. Please either rename, or remove the call to the callback.

Reply:
Oh, I see that we use a similar naming for the other functions that also does more than just validate. IMO that's not very good, as `validate` doesn't not imply any state changes.

Reply:
Just try to follow the original code structure and naming rule.

Reply:
Yeah, I saw that. It is OK as is for now, but IMO should be updated in the future.

Reply:
We need to refactor this one day.